moleculer.gemspec

```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "awesome_print",       "~> 1.8"
   spec.add_dependency "concurrent-ruby",     "~> 1.1"
   spec.add_dependency "ougai",               "~> 1.7"
+  spec.add_dependency "msgpack",             "~> 1.3"
```

Comment:
lets add message pack as a development dependency. that allows people using them gem to choose which serializer (and their libraries) to use instead of making production dependencies install a massive amount of libraries they may not need.

Reply:
Good call.